pkgs/applications/science/logic/surelog/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python3
+, pkg-config
+, libuuid
+, openjdk11
+, gperftools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "surelog";
+  version = "v1.36";
```

Comment:
In the meantime, there is now a v1.37. Which is interesting, because now there is also a corresponding tag `v0.9.1.37` in UHDM, which was previously untagged.